js/script.js

```diff
@@ -0,0 +1,8 @@
+window.onload = async function () {
+  const marquee = new Marquee(CONFIG.marqueeContainer);
+  await marquee.load();
```

Comment:
why use await here? why stop the loading of the page for the marquee? dont' you think it's better to show some loading indicator instead?